### ISSUE TITLE
Missing `WASHING_FLUID` value for check control message

### ISSUE THREAD

**Issue by brave0d** [bimmerconnected/bimmer_connected#547]
### Describe the issue

```
Logger: homeassistant.components.bmw_connected_drive.binary_sensor
Source: components/bmw_connected_drive/binary_sensor.py:252
Integration: BMW Connected Drive (documentation, issues)
First occurred: 20:15:07 (1 occurrences)
Last logged: 20:15:07

'WASHING_FLUID' not an allowed check control message (CheckControlMessage(description_short='WASHING_FLUID', description_long=None, state=<CheckControlStatus.LOW: 'LOW'>))
```

### Expected behavior

adding the correct value when there is a low screen wash level

### Which Home Assistant version are you using?

2023.7.0

### What is your region?

Rest of world

### MyBMW website

- [X] I can still successfully login to the BMW MyBMW website and the car status is available there.
- [X] I have MyBMW enabled for my vehicle.

### Number of cars

- [ ] I have 2 or more cars linked to the MyBMW account.
- [ ] I have a Mini vehicle linked to my account.

**Comment by rikroe** [comment]
Oops, added it to the wrong entity in https://github.com/home-assistant/core/pull/94762... Thanks for the info!

**Comment by rikroe** [comment]
Probably missed the 2023.8 deadline, so will be included in 2023.9 latest...

**Comment by rikroe** [comment]
Closing this as HA 2023.9 was released (including the fix).
